functions/index.js

```diff
@@ -34,28 +40,61 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
         console.log(status);
         console.log(category);
         console.log(createNewTaskSprintNumber);
+        console.log(storyPointNumber);
 
-        db.collection(fullSprintId).doc(category).set({
-                Title: title,
-                Description: des,
-                Priority: priority,
-                Difficulty: difficulty,
-                Creator: creator,
-                Assignee: assignee,
-                ET: estimatedTime,
-                Status: status,
-                Category: category,
-                LogWorkTotalTime: loggedWorkTotalTime,
-                WorkDone: workDone,
-                CreateNewTaskSprintNumber: createNewTaskSprintNumber
+        db.collection("Main").doc("RawData").get().then((doc) => {
+
+                totalDevelopmentTask = doc.data().TotalDevelopmentTask + 1;
+                totalBusinessTask = doc.data().TotalBusinessTask + 1;
+                totalMarketingTask = doc.data().TotalBusinessTask + 1;
+                totalNumberOfTask = (totalDevelopmentTask + totalBusinessTask + totalMarketingTask);
```

Comment:
totalNumberOfTask can be directly fetched from the db

Reply:
okay